packages/app/src/systems/Core/utils/math.ts

```diff
@@ -2,8 +2,9 @@ import type { BNInput } from 'fuels';
 import { bn } from 'fuels';
 import { MAX_FRACTION_DIGITS } from '~/config';
 
-export function formatAmount(amount?: BNInput) {
+export function formatAmount(amount?: BNInput, units?: number) {
   return bn(amount).format({
     precision: MAX_FRACTION_DIGITS,
+    units,
```

Comment:
we don't need this function, can use directly `format` or `formatUnits` from `fuels`

Reply:
Yeah, we should do some refactoring on the code base in the near future.